### PR TITLE
recommission method

### PR DESCRIPTION
**Pull request by jhsenjaliya**

### CODE REVIEW

File: ambariclient/models.py [hunk 1]
```diff
@@ -887,12 +887,23 @@ def execute_action(self, action, context, parameters=None, hosts=None):
         return self.request
 
     def decommission(self, service, hosts):
-        """Decommission slave components on a cluster.
+        self.commission(service, hosts, "decommission")
+
+    def recommission(self, service, hosts):
+        self.commission(service, hosts, "recommission")
+        """Nodemanager recommission does not start nodemanager process on the host. so its required to do it separately"""
+        if service is "YARN":
```
Comment (jimbobhickville) [inline ambariclient/models.py]: Isn't the recommission an asynchronous process?  Shouldn't you wait until the request is finished to submit this `start()` request?  Or is Ambari smart enough to queue it up and wait for the recommission to complete?

Comment (jhsenjaliya) [inline ambariclient/models.py]: @jimbobhickville, didnt noticed that my reply was still pending (until I submitted it just now) , btw, these operations are tested on my end and it works as I described. I also had chance to get it confirmed from HortonWorks guys.

**Comment by jhsenjaliya** [comment]
@jimbobhickville, can you please comment?

**Comment by jimbobhickville** [comment]
@jhsenjaliya - Sorry for the delay. I don't actually work with Ambari any more.  I'll merge this and try to get a release out next week.  If you're interested, I can look into transferring pypi ownership so you can fork this and do releases on your own.

**Comment by jhsenjaliya** [comment]
@jimbobhickville, Sure, I can take care of the release. Please do the needful, I will ask if I need your help in release. Thanks !

**Comment by jhsenjaliya** [comment]
@jimbobhickville or would it be easier to keep the repo here and add me as team member or something so that i can continue with this repo.  however you prefer. pls let me know.

**Comment by jimbobhickville** [comment]
@jhsenjaliya - I have added you as a collaborator. Apologies for being so slow, Holidays and all that. Please email me with this same username at gmail and we can sort out pypi access so you can push releases out as well.

**Pull request merged**